### DISCUSSION TITLE
Open a popup window without side bar and nav bar

### DISCUSSION THREAD

**Discussion by duxonline** [OrchardCMS/OrchardCore#13396] [Q&A] [answered]
Please forgive me if this is a silly question. I'm wondering what's the best way to open a popup window in admin area without side bar and nav bar? Many thank in advance.

**Comment by hishamco** [comment]
We provide a confirm modal OOB, could you please ellaborate what are you trying to do?

Opening a popup windows could use a default bootstrap modal

**Reply by duxonline** [reply]
Thank you @hishamco 

So I have product category: 

![001](https://user-images.githubusercontent.com/10808737/224683515-12127552-2154-46fd-b3fd-bfb86d3a54bb.PNG)

When I click Add button, I would like to open the following popup:

![002](https://user-images.githubusercontent.com/10808737/224683678-27c819af-0634-413f-bee6-2b217bc62df3.PNG)

In the popup I search products, select some of them ,and then click Add button to add selected products to category.

Hopefully it makes more sense to you now.

Thank you again for the help.

**Reply by hishamco** [reply]
You might consider to use `window.open` on your button click https://www.geeksforgeeks.org/javascript-window-open-method/

**Reply by duxonline** [reply]
Yeah, that's the first thing I tried. The problem is I'm not sure what's the best way to hide side bar and nav bar. I guess I can use jquery to hide them, just not sure it's the way we do it in orchard core.

**Reply by hishamco** [reply]
I understand what you mean, you might the unnecessary parts on page. There's another way which is to load on the div that contains the contents that need to be displayed, but the limitation is the content will be unstyled. That's why I prefer the first way or create a new page from scratch to avoid unnecessary parts to show up

**Reply by duxonline** [reply]
You're right. I would like to inherit styles from admin theme and in the meantime I don't need side/nav bar from layout. I will try your first suggestion. Thank you for quick response.

**Comment by ns8482e** [comment, marked as answer]
You can create a Layout that do not have menu/sidebar and use that for popup pages.

**Reply by duxonline** [reply]
Thank you @ns8482e. Since I use popup a lot in the module, a new layout may be the way to go.

**Reply by hishamco** [reply]
> Since I use popup a lot in the module

@fpmatters if you mentioned this at the beginning you will save time :) coz the layout is the good option in this case

**Reply by duxonline** [reply]
You're right. I should have made it clear at the beginning, still learning how to express myself better online. Really appreciate your help.

**Reply by hishamco** [reply]
No problem, the important thing your issue got fixed
